bench/workgen/workgen.cxx

```diff
@@ -305,9 +341,9 @@ int Monitor::run() {
 
     // The whole and fractional part of sample_interval are separated,
     // we don't want to sleep longer than a second.
-    int sample_secs = ms_to_sec(options->sample_interval_ms);
+    int sample_secs = WorkgenTimeStamp::millis_to_sec(options->sample_interval_ms);
```

Comment:
I think we should not use the WorkgenTimeStamp() abstraction except for wiredtiger time stamps.  Other "time" that is already in the code relates to real "wallclock" time.  Just because the current WorkgenTimeStamp() currently uses the system clock doesn't mean it will in the future.  But here (and all previously existing uses of times, I think), we want a real time.  No matter what the underlying implementation of WorkgenTimeStamp, we want to have the monitor thread (for this code) to print every N seconds.

Reply:
Fixed